components/pager/overview.md

```diff
@@ -174,6 +180,8 @@ You can avoid loading all the data at once, as this can be a costly operation. I
 
 ### Two-way Binding
 
+This section shows how to manually implement a behavior that is **built-in** as of version 2.26. See [`InputType`](#features) above.
```

Comment:
This section shows how to two-way bind the `Page` not `PageSize` parameter. This sentence either needs more explanations, or a separate code example.

Reply:
My mistake, thanks.